Sources/Logging/Strings/StoreKitStrings.swift

```diff
@@ -149,6 +159,21 @@ extension StoreKitStrings: LogMessage {
         case .sk2_observing_transaction_updates:
             return "Observing StoreKit.Transaction.updates"
 
+        case let .sk2_unknown_environment(environment):
```

Comment:
All these 5 warnings for untested logs is code paths that could have unit tests 🙏🏻